packages/vite/src/node/plugins/resolve.ts

```diff
@@ -1067,12 +1072,12 @@ export function resolvePackageEntry(
       )
       if (resolvedEntryPoint) {
         debug?.(
-          `[package entry] ${colors.cyan(id)} -> ${colors.dim(
+          `[package entry] ${colors.cyan(idWithoutPostfix)} -> ${colors.dim(
             resolvedEntryPoint,
-          )}`,
+          )} (postfix: ${postfix})`,
```

Comment:
I think we can make this a little cleaner by adding the ` (postfix: ...)` message only when there's a postfix.